lte/gateway/python/magma/pipelined/app/classifier.py

```diff
@@ -0,0 +1,241 @@
+"""
+Copyright (c) 2020-present, Facebook, Inc.
```

Comment:
Please use the new copyright, as used in https://github.com/magma/magma/blob/master/lte/gateway/python/magma/pipelined/bridge_util.py

Reply:
done.